docs/docs/01-ibc/03-apps/00-ibcv2apps.md

```diff
@@ -0,0 +1,307 @@
+---
+title: IBC v2 Applications
```

Comment:
This is very nice! Would it be worth to have a short section on version upgrades? Call out that in v2 you don't need to upgrade a channel anymore, as long as both sides support a version, a packet can be sent.